src/creport.c

```diff
@@ -1056,7 +1056,7 @@ static void cr_find_address(FILE * F, const faction * uf, selist * addresses)
             fprintf(F, "PARTEI %d\n", f->no);
             fprintf(F, "\"%s\";Parteiname\n", f->name);
             if (f->email)
-                fprintf(F, "\"%s\";email\n", f->email);
+                fprintf(F, "\"%s\";email\n", f->email?f->email:"");
```

Comment:
Hier wäre doch faction_getemail() auch gut gewesen, wie du das in src/json.c gemacht hast? Wenn es einen getter gibt (was nicht oft der Fall ist, gebe ich zu), dann sollten wir den benutzen. Wird für die Zukunft nur einfacher.